docs/source/app-dev/authorization.rst

```diff
@@ -157,14 +157,42 @@ the participant user management service *without* issuing new access tokens,
 as would be required for the custom Daml claims tokens.
 
 User access tokens are `JWTs <https://datatracker.ietf.org/doc/html/rfc7519>`_ that follow the
-`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_ with a JSON payload of the following format.
+`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_. There are two
+different JSON encodings: An audience-based token format that relies
+on the audience field to designate that it is designed for a specific
+Daml participant and a scope-based audience format which relies on the
+scope field to designate. Both formats can be used interchangeably but
```

Comment:
```suggestion
different JSON encodings: An audience-based token format that relies
on the audience field to designate that it is defined for a specific
Daml participant and a scope-based audience token format which relies on the
scope field to designate the purpose. Both formats can be used interchangeably but
```

I found this sentence difficult to understand so I tried to rephrase it... not sure mine reads better, but somehow I call for rephrasing ;)